### PR TITLE
Add a config/env setting to set the maximum number of users per room

### PR DESCRIPTION
**Pull request by jeffswartz**

### CODE REVIEW

File: web/js/roomController.js [hunk 1]
```diff
@@ -856,6 +856,21 @@ PhoneNumberController, ResizeSensor */
...
+        if (!maxUsersPerRoom) {
+          return resolve();
+        }
+        return setTimeout(function () {
+          if (numUsrsInRoom > maxUsersPerRoom) {
+            Utils.sendEvent('roomController:meetingFullError');
+            return;
+          }
+          resolve();
+        }, 500);
```
Comment (jeffswartz) [inline web/js/roomController.js]: If anything, I think we can make this shorter than .5 seconds. But I haven't tested enough, and I don't think that's too long to wait for `connectionCreated` events to come in after connecting to the session.

Comment (emilianop11) [reply web/js/roomController.js]: is there any possibility this could take longer than 500 milis?

Comment (emilianop11) [reply web/js/roomController.js]: I tried it locally and it worked perfectly. im asking just in case

File: web/js/roomView.js [hunk 2]
```diff
@@ -88,6 +88,11 @@ BubbleFactory, Clipboard, LayoutManager, $ */
       detail: 'Failed to acquire microphone. This is a known Chrome bug. Please completely quit ' +
               'and restart your browser.',
       button: 'Reload'
+    },
+    meetingFullError: {
+      head: 'Meeting Full',
+      detail: 'This meeting has reached the full capacity of 16 participants. Try&nbsp;joining later.',
```
Comment (rhainer) [inline web/js/roomView.js:94]: Please make the capacity number set by the config variable.

**Pull request merged**